utils/utilities.js

```diff
@@ -56,10 +56,10 @@ module.exports = {
     return sigs;
   },
 
-  sortWalletByAddress(wallets) {
+  sortWalletByAddress(wallets, addressKey = "address") {
     return wallets.sort((s1, s2) => {
-      const bn1 = ethers.utils.bigNumberify(s1.address);
-      const bn2 = ethers.utils.bigNumberify(s2.address);
+      const bn1 = ethers.utils.bigNumberify(s1[addressKey]);
+      const bn2 = ethers.utils.bigNumberify(s2[addressKey]);
```

Comment:
999_benchmark.js needs to be able to sort an array of `JsonRpcSigner` objects that are similar to the array of `Wallet` objects used in test but `JsonRpcSigner` uses `_address` instead of `address` as the key name for the account address.